build-system/tasks/e2e/describes-e2e.js

```diff
@@ -29,7 +29,9 @@ const TIMEOUT = 20000;
  * TODO(estherkim): use this to specify browsers/fixtures to opt in/out of
  * @typedef {{
  *  browsers: (!Array<string>|undefined),
- *  fixtures: (!Array<string>|undefined),
+ *  environments: (!Array<!AmpdocEnvironment>|undefined),
+ *  testUrl: string,
+ *  initialRect: ({{width: number, height:number}}|undefined)
```

Comment:
Cool spec option here! nit: rename initialRect to windowRect or clientRect, initial is implied